### ISSUE TITLE
The warning bell plays when clicking "revert changes" even with all audio cues disabled

### ISSUE THREAD

**Issue by FoxLisk** [microsoft/vscode#199055]
Type: <b>Bug</b>

Just have any changes in a file and click the revert changes button in the source control tab.

The expected result is that when I disable every single audio cue in the app, _the app stops yelling at me_.

VS Code version: Code 1.84.2 (1a5daa3a0231a0fbba4f14db7ec463cf99d7768e, 2023-11-09T10:51:52.184Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i9-13900KS (32 x 3187)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|63.75GB (42.34GB free)|
|Process Argv|--crash-reporter-id eac1c1ba-38b1-4471-aac1-6ee4300e86e7|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
rust-bundle|1Yi|1.0.0
rust-syntax|dus|0.6.1
rust-analyzer|rus|0.3.1740
crates|ser|0.6.3
even-better-toml|tam|0.19.2
vscode-lldb|vad|1.10.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263:30880072
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
3biah626:30602489
f6dab269:30613381
vscrpc:30673769
showlangstatbar:30737416
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
welcomedialog:30887143
pythonnosmt12:30797651
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
dsvsc013:30795093
dsvsc014:30804076
dsvsc015:30845448
pythontestfixt:30871694
pythonregdiag2:30871582
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
dsvsc016:30886110
dsvsc017cf:30886113
dsvsc018cf:30886115
aa_t_chat:30882232

```

</details>

**Comment by lszomoru** [comment]
@hediet, could you please take a look at this one? Thank you!

**Comment by meganrogge** [comment]
I hear no audio cues when I follow these steps. Please try in insider's with all extensions disabled.

**Comment by FoxLisk** [comment]
I dont know what "please try in insider's" means but here is a video showing the bell playing with all extensions disabled

https://github.com/microsoft/vscode/assets/1173225/5eff066f-6b6e-42f8-92f2-1f091e2aefee

**Comment by meganrogge** [comment]
that sound is coming from your operating system, not VS Code.

**Comment by FoxLisk** [comment]
okay. let me rephrase my bug then: VSCode plays an operating system bell when I click discard changes in the git menu.

It's still exactly the same problem! Seems pretty unreasonable to me to close as wontfix because VSCode is generating an OS bell instead of literally playing a bell itself!

**Comment by meganrogge** [comment]
Are you saying you have an issue with the confirmation dialogue that's appearing?

**Comment by FoxLisk** [comment]
My complaint is what it was from the beginning: VSCode, through very normal operation, generates un-mute-able sounds, in particular this one in the git UI. I don't have any particular interest in the implementation details. What I want is for VScode not to make annoying sounds that I cannot turn off.
